readthedocs/rtd_tests/tests/test_doc_building.py

```diff
@@ -40,328 +30,14 @@
 SAMPLE_UTF8_BYTES = SAMPLE_UNICODE.encode('utf-8')
 
 
+# TODO: most of these tests need to be re-written to make usage of the Celery
+# handlers properly to check success/failure
+#
+# NOTE: that they could probably be deleted as well, since we are not using
+# LocalBuildEnvironment in production at all
+@pytest.mark.skip
```

Comment:
Another marker here just to ensure we don't lost this. Another important part that we need before merge.